oxidation/parsec_cli/parsec_sequestre_dump/src/main.rs

```diff
@@ -0,0 +1,176 @@
+                        children: vec![
+                            Data {
+                                name: "file00".into(),
+                                author: alice.device_id.clone(),
+                                version: 1,
+                                created: t0,
+                                updated: t0,
+                                size: 16,
+                                content: "file00's content".into(),
+                                ..Default::default()
+                            },
+                            Data {
+                                name: "folder00".into(),
+                                author: alice.device_id.clone(),
+                                version: 1,
+                                created: t0,
+                                updated: t0,
+                                size: 0,
+                                ..Default::default()
+                            },
+                            Data {
+                                name: "folder01".into(),
+                                author: alice.device_id.clone(),
+                                version: 1,
+                                created: t0,
+                                updated: t0,
+                                size: 34,
+                                children: vec![
+                                    Data {
+                                        name: "file010".into(),
+                                        author: alice.device_id.clone(),
+                                        version: 1,
+                                        created: t0,
+                                        updated: t0,
+                                        size: 17,
+                                        content: "file010's content".into(),
+                                        ..Default::default()
+                                    },
+                                    Data {
+                                        name: "file011".into(),
+                                        author: alice.device_id.clone(),
+                                        version: 1,
+                                        created: t0,
+                                        updated: t0,
+                                        size: 17,
+                                        content: "file011's content".into(),
+                                        ..Default::default()
+                                    }
+                                ],
+                                ..Default::default()
+                            },
+                        ],
+                        ..Default::default()
+                    }
+                ],
+                ..Default::default()
+            })
+        );
+
+        workspace.dump(output).unwrap();
+
+        let loaded_data = Workspace::load(output);
+
+        assert_eq!(workspace, loaded_data);
+
+        std::fs::remove_file(input).unwrap();
```

Comment:
Doing this means the files won't be removed if an assert fails, (this is why fixture with there teardown step always executed is so convenient ^^)